src/resources/scss/components/_card.scss

```diff
@@ -27,4 +27,45 @@
 
 .card > .card-body + .list-group > .list-group-item:first-of-type {
     border-top: 1px solid $list-group-border-color;
-}
\ No newline at end of file
+}
+
+.card {
+    background-color: white;
+}
+
+.card .card-header {
+    position: relative;
+    padding: .75rem 1.25rem;
+    margin-bottom: 0;
+    background-color: rgba(0,0,0,0.03);
+    border-bottom: 1px solid rgba(0,0,0,0.125);
+}
+.card .card-header h3{
+    color: #fff;
+    font-weight: 400;
+}
+.card a {
+    color: #333;
+    text-decoration: none;
+}
+
+.card:hover .card-image img {
+    width: 110%;
```

Comment:
Why `width: 110%;`?

Reply:
Mouse over will have a magnifying effect

Reply:
Ahhh - sorry, didn't see that `:hover`. 👍